__tests__/app.test.js

```diff
@@ -251,3 +251,25 @@ describe("GET /api/articles", () => {
     });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments for the article given by article_id, each comment contains comment_id, votes, created_at, author and body as properties", () => {
+    const article_id = 5;
+    return request(app)
+      .get(`/api/articles/${article_id}/comments`)
+      .expect(200)
+      .then((res) => {
+        const { comments } = res.body;
+        expect(comments).toBeInstanceOf(Array);
```

Comment:
When we are testing against each of the items in an array, we need to check that the length of the array is what we're expecting before we enter the forEach. Otherwise, if the array is empty (but still an array), it will pass through the forEach because it will never find an object to return false against.